### ISSUE TITLE
getPhpUnitXmlDir() cause an error on static::bootKernel();

### ISSUE THREAD

**Issue by knyk** [liip/LiipFunctionalTestBundle#386]
`KernelTestCase::getPhpUnitXmlDir()` method is removed since 4.0 from `symfony/framework-bundle` and cause an error in tests where `static::bootKernel()` is called.

**Comment by Jean85** [comment]
The `bootKernel()` method is not part of this bundle; we are anyway targeting support for Symfony 4 on a new version, 2.0. You can already test it with the [alpha release](https://github.com/liip/LiipFunctionalTestBundle/releases/tag/2.0.0-alpha1).

**Comment by knyk** [comment]
yes, `bootKernel()` is not, but `WebTestCase::getKernelClass` is.

it uses removed method `KernelTestCase::getPhpUnitXmlDir()` and that's why error is triggered.

bundle version 1.9 can be installed on Symfony 4 so i think it is not only symfony related issue

**Comment by Jean85** [comment]
The issue then is that the 1.x tags are installable with Symfony 4, they shouldn't.

**Comment by fvozar** [comment]
Any updates on this issue? Do anyone know a working workaround?

**Comment by rufinus** [comment]
@fvozar `"liip/functional-test-bundle": "~2.0@dev",`

**Comment by kedramon** [comment]
Simple override works
```
protected static function getKernelClass(): string
    {
        return \App\Kernel::class;
    }
```

**Comment by nietonfir** [comment]
Setting `<server name="KERNEL_DIR" value="app" />` in `phpunit.xml` works just fine as well.

**Comment by mtnorthrop** [comment]
Nietonfir's suggestion works for getting rid of the deprecation notices in Symfony 3.4 as well.

**Comment by magnetik** [comment]
I think it's fixed since 8588dbd9fa78e49dfb135fd1127b2681914065da :)
